### ISSUE TITLE
Button accessibility updates

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4854]
When updating links, I found some links that should be buttons and vice-versa.  Opening up a new issue to review buttons afterwards (both those identified during the link work, and those already listed as buttons, as some probably need to be links instead).

this work needs to wait until #4221 is complete.
